cmd/root.go

```diff
@@ -32,17 +32,20 @@ type Root struct {
 }
 
 var debugMode bool
+var EnableASNIColor bool
```

Comment:
It's "ANSI" not "ASNI", correct it everywhere

Reply:
Corrected